src/Utils/BlockTemplateUtils.php

```diff
@@ -137,6 +140,9 @@ public static function gutenberg_build_template_result_from_file( $template_file
 		$template->has_theme_file = true;
 		$template->is_custom      = false; // Templates loaded from the filesystem aren't custom, ones that have been edited and loaded from the DB are.
 		$template->post_types     = array(); // Don't appear in any Edit Post template selector dropdown.
+		if ( 'wp_template_part' === $template_type ) {
+			$template->area = 'uncategorized';
+		}
```

Comment:
With this code, all template parts will have `area` set to `uncategorized`. This is fine for now. But I think we need to add a todo here to improve this in the future. Each template part may have a different area, and we can filter that `area` base on the `slug` IMO.